test/test_lib/utils.js

```diff
@@ -335,6 +335,12 @@ Utils.prototype = {
     '0x0000000000000000000000000000000000000000',
 
   NULL_ADDRESS: '0x0000000000000000000000000000000000000000',
+
+  SENTINEL_ADDRESS: '0x0000000000000000000000000000000000000001',
+
+  MAX_FUTURE_END_HEIGHT: new BN(
```

Comment:
let's rename this `MAX_UINT256` then? assuming this is 
```
uint256 public constant MAX_FUTURE_END_HEIGHT = uint256(
        0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff);
```

Reply:
Done